scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -85,63 +87,6 @@ workflow GvsQuickstartIntegration {
     }
 }
 
-task Setup {
```

Comment:
This is almost completely a cut/paste of a task from the quickstart integration test to `GvsUtils.wdl` so it could be reused.